### PR TITLE
Post List: Fix the stretched thumbnail images.

### PR DESCRIPTION
**Pull request by pablinos**
When we take the thumbnail image from the post content, it might not be
the correct aspect ratio so, as it is displayed in a square, it appears
stretched and distorted.

#### Changes proposed in this Pull Request:

This fixes that by applying `object-fit` to the image. This CSS isn't
supported in IE, but I believe we're no longer supporting IE, and it's a
visual glitch, that doesn't otherwise break the experience.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Enable the post list enhancements
* Create a post with no set featured image, and a rectangular image in the content
* See that the image is stretched when displayed in the post list
* With this branch reload the post list and see that the image is cropped rather than stretched

| Before | After |
|--------|------|
|image|image|

### CODE REVIEW

**Review by jeherve — DISMISSED** [review]

File: projects/packages/post-list/src/style.css [hunk 1]
```diff
@@ -8,6 +8,7 @@
 	width: 50px;
 	height: 50px;
 	border-radius: 4px;
+	object-fit: cover;
```
Comment (Rolilink) [inline projects/packages/post-list/src/style.css:11]: Just wanna say I  didn't know this existed really cool will add it to my CSS note, also It seems you are correct and we don't support IE (https://fieldguide.automattic.com/browser-support/)

**Review by Rolilink — APPROVED** [review]

**Review by retrofox — APPROVED** [review]
LGTM

File: projects/packages/post-list/src/style.css (hunk 1 above)
Comment (Rolilink) [inline projects/packages/post-list/src/style.css:11]: Just wanna say I  didn't know this existed really cool will add it to my CSS note, also It seems you are correct and we don't support IE. (PCYsg-2bz-p2)

Comment (retrofox) [reply projects/packages/post-list/src/style.css:11]: same :-)

Comment (pablinos) [reply projects/packages/post-list/src/style.css:11]: Interestingly, we do support IE in the Classic Dashboard, which is what this affects. It's probably a visual glitch that can be lived with for now.

**Comment by pablinos** [comment]
I lost the approval, as I needed to update the package version. If someone could approve it again, that would be great!

**Review by retrofox — APPROVED** [review]

**Pull request merged**